tst/EnergyPlus/unit/CMakeLists.txt

```diff
@@ -212,18 +203,27 @@ set( test_src
   ZoneTempPredictorCorrector.unit.cc
   main.cc
 )
+
 set( test_dependencies
   energyplusapi
- )
+)
+
+if(LINK_WITH_PYTHON)
+  if (CMAKE_HOST_UNIX)
+    find_package(PythonLibs 3 REQUIRED)
+    include_directories(${PYTHON_INCLUDE_DIRS})
+    list(APPEND test_src api/datatransfer.unit.cc)
+    list(APPEND test_src PluginManager.unit.cc)
+    list(APPEND test_dependencies ${PYTHON_LIBRARIES} )
+  endif()
+endif()
```

Comment:
There is some voodoo going on, and I'm not investigating further for now.  On Linux/Mac, the API code is being unit tested thoroughly.  On Windows it doesn't work.  And being very frank, I'm not quite sure how it is actually working on Mac/Linux.  In any case, I'm limiting it to just Mac/Linux and we'll figure it out later.  The most likely scenario is that we don't try to test any API code from the unit test binary, and instead we do the API calls from the separate API test scripts/programs.